.github/workflows/dockerhub-readme.yml

```diff
@@ -0,0 +1,64 @@
+name: Generate DockerHub READMEs
+
+'on':
```

Comment:
```suggestion
on:
```

Are the quotes here needed? I've never had to use quotes for the `on` key when doing GitHub Actions ([src1](https://github.com/rapidsai/docker/blob/branch-0.17/.github/workflows/dockerfiles.yaml#L3), [src2](https://github.com/rapidsai/gpuci-scripts/blob/branch-0.17/.github/workflows/jenkins_plugins.yaml#L3)).

Reply:
Yeah I found that depending on how the yaml is deserialized, `on` may be converted to `true`.

E.g 
<img width="1792" alt="image" src="https://user-images.githubusercontent.com/68436579/101640083-ac92ba80-39f5-11eb-910e-27b382603e68.png">

But I have unquoted it though.